shared/src/jvmMain/kotlin/com/mbta/tid/mbta_app/ProjectUtils.kt

```diff
@@ -62,7 +63,7 @@ object ProjectUtils {
     }
 
     fun fetchTestData(): Unit = runBlocking {
-        startKoin { modules(appModule(AppVariant.Prod), platformModule) }
+        startKoin { modules(appModule(AppVariant.Staging), platformModule) }
```

Comment:
question(non-blocking): Why not continue to use Prod?

Reply:
Prod doesn’t have https://github.com/mbta/mobile_app_backend/pull/323 so the global data can’t load.

Reply:
Ah! Good reminder that we need to kick off a backend deploy anyway before the next release, I can kick that off.